perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -61,53 +155,37 @@ def GetInfo():
 def Prepare(benchmark_spec):
   """Prepare the virtual machine to run FIO.
 
-     This includes installing fio, bc. and libaio1 and insuring that the
-     attached disk is large enough to support the fio benchmark.
+     This includes installing fio, bc, and libaio1 and insuring that
+     the attached disk is large enough to support the fio
+     benchmark. We also make sure the job file is always located at
+     the same path on the local machine.
 
   Args:
     benchmark_spec: The benchmark specification. Contains all data that is
         required to run the benchmark.
+
   """
-  vms = benchmark_spec.vms
-  vm = vms[0]
+
+  if (FLAGS.fio_jobfile and
+      (FLAGS.against_device or FLAGS.device_fill_size or FLAGS.io_depths)):
+    logging.warning('Fio job file specified. Ignoring other fio options.')
```

Comment:
Give more detailed logging here, current message is a bit confusing.
eg, job_file= ..., against_device=...,